_build_cfg.yml

```diff
@@ -210,6 +208,8 @@ Topics:
     File: syncing_groups_with_ldap
   - Name: Available Ports
     File: available_ports
+  - Name: Aggregating Logs with the EFK Stack
```

Comment:
Can we drop "with the EFK Stack" in the title?

Also, the "Aggregating Container Logs" title was more specific as to what types of logs the topic is referring to, since there's other OpenShift service/platform logs that it could also be referring to.